plugins/pur/core/pom.xml

```diff
@@ -329,7 +328,7 @@
     <dependency>
       <groupId>org.apache.jackrabbit</groupId>
       <artifactId>jackrabbit-core</artifactId>
-      <version>${jackrabbit-core.version}</version>
+      <version>${jackrabbit.version}</version>
```

Comment:
Please use dependency management in the parent POM.
All references to versions should only exist in the parent POM.